yarn.lock

```diff
@@ -2232,6 +2243,16 @@
     fs-extra "^9.0.0"
     glob "^7.1.6"
 
+"@sewing-kit/core@^0.4.0":
```

Comment:
Hmm, do you know what is bringing in the sewing-kit/core@0.1.4 dependency? It would be nice to be able to dedupe these.

Reply:
Well, huh. I ran [`yarn-deduplicate`](https://github.com/atlassian/yarn-deduplicate) but it didn't dedupe `@sewing-kit/core`